<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pB_start">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_time">
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Время</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="tB_text">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pB_round">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Круг</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pB_clear">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pB_clear</sender>
   <signal>clicked()</signal>
   <receiver>tB_text</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pB_clear</sender>
   <signal>clicked()</signal>
   <receiver>l_time</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
